functions/lib/integration/import-product.js

```diff
@@ -152,9 +146,11 @@ module.exports = ({ appSdk, storeId, auth }, tinyToken, queueEntry, appData, can
                 if (productId) {
                   method = 'PATCH'
                   endpoint = `/products/${productId}.json`
-                } else {
+                } else if (!productId && (tipo !== 'estoque' || !tipo)) {
```

Comment:
"se 1 === 1, se não se 1 !== 1"...
Não precisa checar `!productId` aqui:

```suggestion
                } else if (tipo !== 'estoque' || !tipo) {
```

Fora isso, não deveria adicionar o produto somente se `tipo === 'produto'` (mais estrito) não? 🤔